css/_project-custom.scss

```diff
@@ -1862,3 +1836,24 @@ li {
     @include u-margin-top(1);
   }
 }
+
+p + .usa-accordion,
+h2 + *,
+h3 + *,
+h4 + *,
+h5 + *,
+h6 + * {
+  margin-top: units(2);
```

Comment:
This code seems to impact the top heading margins (1.5em) when another heading element follows it.  See "Spacing" https://federalist-proxy.app.cloud.gov/preview/uswds/uswds-site/dw-remove-bourbon/components/typography/#typesetting